### DISCUSSION TITLE
About cauldron

### DISCUSSION THREAD

**Discussion by Sanbursess** [Roadhog360/Et-Futurum-Requiem#121] [Ideas]
When exposed to [rain](https://minecraft.fandom.com/wiki/Rain), [cauldrons](https://minecraft.fandom.com/wiki/Cauldron) have a possibility to slowly filling with water.
I think this is a good idea for skyblock.

**Comment by makamys** [comment]
Doesn't vanilla 1.7.10 already have this? According to the wiki pages you linked this was added in 1.3.1.

**Reply by Sanbursess** [reply]
Sorry，I don't notice that, May be The probability of filling is very small. So I request to add a config to configure it.

**Reply by makamys** [reply]
The probability doesn't look different in 1.16.5. In both versions there's a 1/16 chance every tick to do a "snow/ice"-type tick on the top block of a random column in each chunk, and if the cauldron got selected for this then there's a 1/20 chance its water level will get incremented.

**Reply by Sanbursess** [reply]
My fault! I don't know what's matter.
May be I'm not lucky, I tried in1.18.2 and 1.7.10, probably ten minutes in the rain, the cauldron is no water.(I used two cauldrons),
or I used cauldron was not enough.
But today, just a few minutes in1.7.10, nine cauldrons, one of the cauldron already has a little water.
Just one! another cauldrons still empty

**Comment by Roadhog360** [comment]
I'm unsure if configuring base game features is within the scope of Et Futurum Requiem unless it is related to what I am adding.

**Reply by Sanbursess** [reply]
Sure, That's a request. If you don't mind, and you are not busy. I wish you can help me.
If you will do that, don't forget the lava and water in cauldron is able to use bucket  to take out.
Perhaps you can only make the water is able to use bucket  to take out.
And one more thing, the recipe of etfuturum:nether_brick_wall have a little mistake. The recipe output one nether brick wall, it should be six.
Et_Futurum_Requiem-2.3.1

**Comment by Roadhog360** [comment]
Are you sure you are on 2.3.1? I fixed the wall outputs twice.

**Reply by Sanbursess** [reply]
![2022-05-06 181905](https://user-images.githubusercontent.com/104017558/167113768-2c74d5a0-f289-4e1f-afeb-6eb983d0ce89.png)
![2022-05-06 181945](https://user-images.githubusercontent.com/104017558/167113794-ac9ede1d-435b-492e-9992-d1b5da15deff.png)
Unfortunately, I redownload and I'm sure version is 2.3.1, no beta, Uploaded: Dec 14, 2021, and I use CraftTweaker to fix it.

**Reply by Roadhog360** [reply]
Time to fix it again... *sigh* I think some commits aren't being merged properly causing it to repeatedly be "unfixed"
Third time's the charm, I guess.

**Reply by Sanbursess** [reply]
Good luck!
